appveyor-linux.yml

```diff
@@ -0,0 +1,45 @@
+version: 1.0.{build}
+image: ubuntu1804
+configuration: Release
+environment:
+  git_access_token:
+    secure: FxcQ9C8a/NgcQB5dFdZts6ZWEDT4zMhA4qPQAYwWc7huMmhmTIl1sbFEIaAWQMTL
+  matrix:
+    - framework: netcoreapp2.0
+
+before_build:
+- sh: >-
+    sudo apt-get -y install gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget xvfb
+
+    dotnet restore lib/PuppeteerSharp.sln
+
+    dotnet dev-certs https -ep lib/PuppeteerSharp.TestServer/testCert.cer
+
+    sudo openssl x509 -inform der -in lib/PuppeteerSharp.TestServer/testCert.cer -out /usr/local/share/ca-certificates/testCert.crt -outform pem
+
+    sudo update-ca-certificates
+
+build_script:
+- sh: >-
+    dotnet build -f $framework ./lib/PuppeteerSharp.Tests.CloseMe/PuppeteerSharp.Tests.CloseMe.csproj
+
+    dotnet build -f $framework ./lib/PuppeteerSharp.DevicesFetcher/PuppeteerSharp.DevicesFetcher.csproj
+
+    dotnet build -f $framework ./lib/PuppeteerSharp.Tests.DumpIO/PuppeteerSharp.Tests.DumpIO.csproj
+
+    dotnet build -f netstandard2.0 ./lib/PuppeteerSharp.TestServer/PuppeteerSharp.TestServer.csproj

```

Comment:
should this line also use `$framework` ?

Reply:
Hi. As the project was set to netstandard2.0 it was impossible to compile it with netcoreapp2.0.

Reply:
@Meir017 suggested to create a new var for instance
$netcoreapp and $netstandard
If we just want to change everything to $framework we need to modify the projectfile (if I undestand it correctly) for ./lib/PuppeteerSharp.TestServer to use the same frameworks as the other projects. With .netcoreapp2.0 it did not compile because it was not set in the project file. I did not try to change to the multi framework approach the other projects are using.

Reply:
I think we are good with `$framework` we are consistent with the other appveyor file.
Yeah, I know that bumping the version won't be as easy as changing one variable. But if we know that the only thing we need to do in appveyor is changing the matrix configuration that would be good for us.

Reply:
```suggestion
    dotnet build ./lib/PuppeteerSharp.TestServer/PuppeteerSharp.TestServer.csproj
```